apis/v1alpha1/generator.yaml

```diff
@@ -346,3 +346,12 @@ resources:
     fields:
       Name:
         is_primary_key: true
+  DBProxy:
+    exceptions:
+      terminal_codes:
+        - DBProxyAlreadyExistsFault
```

Comment:
Hmm, not sure we need to list this one. This should only ever appear if the ACK runtime is seriously broken and the initial call to `ReadOne` for `DescribeDBProxies` returns NotFound by mistake and we try to create a DB proxy that already exists. I think it's actually safer to leave this one off the list of terminal conditions.